src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,96 @@
-/* eslint-disable */
 import React from 'react';
+import { useAppSelector } from '../../app/store';
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
+import { Status } from '../../types/Status';
```

Comment:
Ensure that the `Status` enum is correctly defined in the `../../types/Status` module. It should include values like `Completed` and `Active` to match the filtering logic.